src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/KeyworkerMigrationService.java

```diff
@@ -44,9 +44,8 @@ public void migrateKeyworkerByPrison(final String prisonId) {
         offenderKeyworkerRepository.saveAll(translate(allocations));
 
         // Mark prison as migrated
-        repository.findById(prisonId).ifPresent(prison -> {
-            prison.setMigrated(true);
-            prison.setMigratedDateTime(LocalDateTime.now());
+        repository.findByPrisonCode(prisonId).ifPresent(prison -> {
+            prison.setEnabled(true);
```

Comment:
This doesn't set the migrated date time anymore. It's highly unlikely this code will be used again but I guess best to retain existing functionality